fineract-provider/build.gradle

```diff
@@ -853,7 +853,7 @@ generateGitProperties.outputs.upToDateWhen { false }
 
 //This is for swagger code generation
 dependencies {
-    swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.20'
+    swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.21'
```

Comment:
@vorburger 
It turns out (for some reason) that `./gradlew generateSwaggerCode` only works for version <= `3.0.18` and not above. This is a bug in swagger codegen https://github.com/swagger-api/swagger-codegen/issues/10157 

So for now, we'll have to set it to `3.0.18`. Also, we should set up .travis.yml as soon as possible so that it can catch these problems for us.

Reply:
> we should set up .travis.yml as soon as possible so that it can catch these problems for us.

Would you like to contribute a PR doing so?

Reply:
Yes sure, I'll setup the .travis.yml and raise a PR.
Just to clarify, we want add 1. generate and 2. build to the .travis.yml file right?